katas/chicken-sexing/kevin/chickenSexing.js

```diff
@@ -1,3 +1,17 @@
 export function correctness(bobsDecisions, expertDecisions) {
 
-}
\ No newline at end of file
+    let points = 0;
+
+    for (let i = 0; i <= bobsDecisions.length - 1; i++) {
+        if (bobsDecisions[i] == expertDecisions[i]) {
+            points = points + 1;
+
```

Comment:
Please remove remove empty lines since those do not seem to have a purpose in this code